src/main/java/net/masterthought/cucumber/generators/FailuresOverviewPage.java

```diff
@@ -16,6 +21,19 @@ public String getWebPage() {
 
     @Override
     public void prepareReport() {
-        context.put("all_features", report.getAllFeatures());
+        List<Element> failures = new ArrayList<>();
+        for (Feature feature : report.getAllFeatures()) {
+            if (feature.getStatus().isPassed()) continue;
+
+            for (Element element : feature.getElements()) {
+                if (!element.getBeforeStatus().isPassed()
+                    || !element.getStepsStatus().isPassed()
+                    || !element.getAfterStatus().isPassed()) {
```

Comment:
In that case I would expect that the report should have scenario failed as well. What I'm trying to say is that the report generation should not change the result stored in json file.

Waiting for updated PR